types/pomelo/index.d.ts

```diff
@@ -0,0 +1,890 @@
+declare module pomelo {
+
+    /**
+     * Application
+     */
+    export class Application {
+        /**
+         * Get application base path
+         *
+         *  // cwd: /home/game/
+         *  pomelo start
+         *  // app.getBase() -> /home/game
+         *
+         * @return {string} application base path
```

Comment:
Don't add types in jsdoc comments, the code is already a type definition after all.